tsconfig.json

```
@@ -1,21 +1,36 @@
 {
   "compilerOptions": {
     "target": "ESNext",
-    "useDefineForClassFields": true,
     "lib": ["DOM", "DOM.Iterable", "ESNext"],
-    "types": ["vite/client", "vite-plugin-svgr/client", "mocha", "chai"],
+    "types": ["vite/client", "vite-plugin-svgr/client"],
+    "module": "ESNext",
```

Let's stick to lower case, as documented
```suggestion
    "module": "esnext",
```